stack_orchestrator/data/compose/docker-compose-fixturenet-laconicd.yml

```diff
@@ -15,12 +15,12 @@ services:
       - "6060"
       - "26657"
       - "26656"
-      - "9473"
+      - "9473:9473"
```

Comment:
This seems indicative of some problem somewhere?

Reply:
Yes, it is.  The web console doesn't track the auto-assigned ports, so unless it is 9473 the UI is broken.  We'll need to fix the console.